Fastlane/Fastfile

```diff
@@ -34,8 +34,8 @@ lane :bump_version do |options|
 
     # Bump version number
     increment_version_number_in_xcodeproj(bump_type: component, target: "Movapp (iOS)")
-    increment_version_number_in_xcodeproj(bump_type: component, target: "WatchMovapp")
-    increment_version_number_in_xcodeproj(bump_type: component, target: "WatchMovapp WatchKit Extension")
+    #increment_version_number_in_xcodeproj(bump_type: component, target: "WatchMovapp")
```

Comment:
Tohle můžeš příště klidně zrovna smazat, kdybychom se k tomu vraceli, vylovíme to z historie.

Reply:
ok :)